src/erf.jl

```diff
@@ -92,7 +92,7 @@ end
 
 function erf(x::Float64)
     ix = get_high_word(x)
-    sign = ix>>31
+    sign = Int(ix>>31)
```

Comment:
be sure you test this kind of thing on both 32 and 64 bit - probably easiest to turn on appveyor